core/src/blocktree.rs

```diff
@@ -765,9 +793,9 @@ impl Blocktree {
     }
 
     fn is_detached_head(meta: &SlotMeta) -> bool {
```

Comment:
"detached_heads" is possibly a bad naming choice, and here it looks to consume a pub fn that's the opposite sense

perhaps "orphans" would be better, i.e. slots without parents